sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnAdapters.java

```diff
@@ -47,10 +51,10 @@ private DoFnAdapters() {}
     }
   }
 
-  /** Create a {@link OldDoFn} that the {@link DoFn}. */
+  /** Creates an {@link OldDoFn} that delegates to the {@link DoFn}. */
```

Comment:
Should this class (and relevant methods) be `@Deprecated` with appropriate comments indicating we should stop using them?